jdbcSource/README.md

```diff
@@ -0,0 +1,234 @@
+    try {  
+        // Creating a map of each row, used to INSERT into our type
+        var resultData = {}
+        resultData.id = message.id
+        resultData.age = message.age
+        resultData.first = message.first
+        resultData.last = message.last
+
+        // Inserting the map into our JDBCType
+        INSERT JDBCType(resultData)
+    } catch (error) {
+        // Catching any errors and throwing the exception.
+        exception(error.code, error.message)
+    }
+    ```
+
+## Messages from the Source
+
+Messages that are sent to the source as Notifications from the pollQuery are JSON objects in the following format:
+```
+{
+   {columnName:columnValue, columnName:columnValue, etc..}
+}
+```
+The data is formatted as a HashMap which represents a row of data. Each map is a series of key-value pairs with the keys 
+being the column name and the values being the column value. If multiple rows of data are returned by the pollQuery, each row
+will be sent as a unique Notification. 
+
+*   (**NOTE:** Multiple rows of data sent as Noticfications will be returned in a random order.)
+
+## Select Statements
+
+In order to interact with the JDBC Source, one option is to use VAIL to select from the source. To do this, you will need 
+to specify the SQL Query you wish to execute against your database as part of the WITH clause. The SQL Queries used here must 
+only be **SELECT STATEMENTS**. The data will be returned as an array of the resulting rows. The following is an 
+example of a Procedure created in VANTIQ Modelo querying against a JDBC Source.
+
+```
+PROCEDURE queryJDBC()
+
+try {        
+    // Normal SELECT Statement in VAIL, but using WITH Clause is important.
+    SELECT * FROM SOURCE JDBC1 AS results WITH
+    // WITH Clause specifies the query, whose value is the SQL Query.
+    // This parameter must be named 'query'.
+    query: "SELECT id, first, last, age FROM Test" 
+
+    {
+        // Iterating over each row of data in 'results'
+        FOR (row in results) {
+            // Creating a map of each row, used to INSERT into our type
+    	    var resultData = {}
+    	    resultData.id = row.id
+    	    resultData.age = row.age
+    	    resultData.first = row.first
+    	    resultData.last = row.last
+            
+	    // Inserting the map into our JDBCType
+    	    INSERT JDBCType(resultData)
+        }
+    }
+} catch (error) {
+    // Catching any errors and throwing the exception.
+    exception(error.code, error.message)
+}
+```
+
+## Publish Statements
+
+Another method to interact with the JDBC Source is to use VAIL to publish to the source. To do this, you will need to
+specify the SQL Query you wish to execute against your database as part of the Publish Parameters. The SQL Queries used here 
+can be **CREATE**, **INSERT**, **DELETE**, **DROP**, or other commands supported by your SQL database. The following is an 
+example of a Procedure created in VANTIQ Modelo publishing to a JDBC source.
+
```

Comment:
an example of an INSERT would be good here.
For example, can you pass an object to the INSERT with the values to be inserted?
Or do you have to create a query string with the full INSERT text??